Wikipedia/View Controllers/WebView/NativeSectionTitleOverlays/WMFSectionTitlesViewController.m

```diff
@@ -0,0 +1,231 @@
+//  Created by Monte Hurd on 7/31/15.
+//  Copyright (c) 2015 Wikimedia Foundation. Provided under MIT-style license; please copy and modify!
+
+#import "WMFSectionTitlesViewController.h"
+#import "WMFTitleOverlayLabel.h"
+#import "WMFTitleOverlayModel.h"
+#import "NSString+Extras.h"
+#import "UIView+WMFSearchSubviews.h"
+#import <Masonry/Masonry.h>
+
+@interface WMFSectionTitlesViewController ()
+
+@property (nonatomic, strong) NSMutableArray* models;
```

Comment:
"models" is a bit generic… better to name them for their purpose… like overlayModels or overlayViewModels.